demos/button.html

```diff
@@ -156,7 +166,34 @@ <h1 class="mdc-typography--display2">Ripple Enabled</h1>
               Primary
             </button>
             <button class="mdc-button mdc-button--raised mdc-button--accent">
-              Accent
+              Secondary
+            </button>
+          </div>
+        </fieldset>
+
+        <fieldset>
+          <legend class="mdc-typography--title">Unelevated Button</legend>
+          <div>
+            <button class="mdc-button mdc-button--unelevated">
+              Baseline
+            </button>
+            <a href="javascript:void(0)" class="mdc-button mdc-button--unelevated">
```

Comment:
Note: "Link" buttons were moved to the end of these lines in #1176, and "Disabled" buttons were removed (replaced by a checkbox at the top of the demo).